### ISSUE TITLE
Users and Groups

### ISSUE THREAD

**Issue by MenthaSuaveolens** [linuxmint/cinnamon-control-center#301]
```
 * cinnamon-control-center version
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) Linux Mint 21 Cinnamon
 * Graphics hardware *and* driver used
 * 32 or 64 bit
 ```

 When the account-name contains an ampersand, it is displayed incorrectly in the dialog Users and Groups

Create an account, eg. User1 & User2

Account should be visualized properly

As one can notice in this screenshot. The language was set to English (see Dialog beneath), but the language of the users and groups dialog is still in the installed language (Dutch). The second user is visualized the same as the first one.

![UsersAndGroups](https://user-images.githubusercontent.com/47174957/190895641-8f4de622-15f8-444e-980e-34cc001105a8.png)
tion**

**Comment by MenthaSuaveolens** [comment]
Don't try this on a working system! It changed the log-in-password of the main user

**Comment by MenthaSuaveolens** [comment]
Fixed in 21.3
